nixos/modules/services/audio/ympd.nix

```diff
@@ -0,0 +1,62 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.ympd;
+in {
+
+  ###### interface
+
+  options = {
+
+    services.ympd = {
+
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        description = "Whether to enable ympd, the MPD Web GUI.";
+      };
+
+      webPort = mkOption {
+        type = types.int;
```

Comment:
The `--webport` of ympd can actually take much more values, including the address to listen on and can even do TLS with `"ssl://8081:/path/to/ssl.pem"`, see https://github.com/notandy/ympd#ssl-support. I think we should support this. Should we maybe change this to a string and keep the naming like upstream?